### ISSUE TITLE
Dochian Channels With mpl_Finance

### ISSUE THREAD

**Issue by DrChandrakant** [matplotlib/mplfinance#596]
Those Who Looks For Dochian Channel with mpl_finance

![NIFTY_2023-03-09_14-09-42](https://user-images.githubusercontent.com/127198654/223967052-b35bc855-204b-4111-aec7-6531ba9e2490.png)

```
import OfflineData # Function Fetch Data From Mysql Stored Candle
import pandas_ta as ta
import mplfinance as mpl
```

```
df = OfflineData.getStockData('tcs')
df.tail(2)
```

<html>
<body>

Id | Open | High | Low | Close | Volume
-- | -- | -- | -- | -- | --

348.0 | 3361.9 | 3404.95 | 3359.00 | 3378.0 | 1333840.0
349.0 | 3364.0 | 3396.00 | 3330.85 | 3390.4 | 1844590.0

</body>
</html>

```
donchiandf = ta.donchian(df["High"], df["Low"], lower_length=10, upper_length=15)
donchiandf.tail(2)

DCU = donchiandf[['DCU_10_15']]
DCM = donchiandf[['DCM_10_15']]
DCL = donchiandf[['DCL_10_15']]

apds = [
    mpl.make_addplot(DCU,color='#2962FF',panel=0,),
    mpl.make_addplot(DCM,color='#FF6D00',panel=0,),
    mpl.make_addplot(DCL,color='#2962FF',panel=0,),
]

mpl.plot(
    df,
    volume=True,
    type="candle",
    style="yahoo",
    addplot=apds,
    figscale=1.2,
    figsize=(46,26,)
)
```
![dochian](https://user-images.githubusercontent.com/127198654/223966251-1ad7c62f-70d1-4fc7-a823-694f8432fcef.png)

**Comment by alexpvpmindustry** [comment]
cool, but i dont think is considered an enhancement.

**Comment by DrChandrakant** [comment]
> cool, but i dont think is considered an enhancement. do you mean to have this implemented in mplfinance?

This Guidance Code for People who Search For It Should Be Part of the Example section.

**Comment by DrChandrakant** [comment]
> ok then maybe this should not be open... its not an issue. maybe a community/example notebook? you want to help create one? @DrChandrakant

Yes I Want to Add This And Many More Indicator I have Prepared For the community/example notebook

**Comment by alexpvpmindustry** [comment]
u know how to make a pull request in github?

**Comment by DrChandrakant** [comment]
Okay I Will Try It pull request
